lib/krane/errors.rb

```diff
@@ -1,8 +1,9 @@
 # frozen_string_literal: true
+require 'kubernetes-deploy/errors'
+
 module Krane
-  class FatalDeploymentError < StandardError; end
-  class FatalKubeAPIError < FatalDeploymentError; end
```

Comment:
Why remove the `Krane` version of this one?

Reply:
Sorry, it was a bad diff/rebase! Thanks for catching it 👍